### ISSUE TITLE
Tracking Indonesian Translation Progress (M4)

### ISSUE THREAD

**Issue by girikuncoro** [kubernetes/website#22296]
This is tracking issue of our progress from Q3 2020 onwards. In the beginning of this milestone, as per [our metrics](https://app.circleci.com/pipelines/github/jk8s/sig-docs-id/31/workflows/104350b4-55f7-4447-9b7c-ea080f1bca8d/jobs/52), we have translated 150 pages from 60 contributors, with around 8,000 views per month.

## Maintainers
* @ariscahyadi
* @danninov
* @girikuncoro
* @irvifa
* @phanama
* @qbl
* @wahyuoi

## For New Contributors
To translate a page:
1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the name of the page you would like to translate. Take only one page at a time.
3. Fork this repo, make your translations, and submit a pull request with `language/id` label!
4. After submit, reviewers will give comments in couple of days, please address them. It's recommended to also [squash](https://github.com/wprig/wprig/wiki/How-to-squash-commits) your commits with good commit message.

### TL;DR Top 5 Mistakes for New Contributors
1. We use "kamu" instead of more formal "Anda" to be more friendly with our readers.
2. Use CamelCase style to write Kubernetes API objects, see complete list [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/).
    - **Correct:** PersistentVolume. **Incorrect:** persistent volume, `PersistentVolume`, persistentVolume
    - **Correct:** Pod. **Incorrect:** pod, `pod`, "pod"
3. Italicize imported english words if you can't find the indonesian words.
    - **Correct:** _controller_. **Incorrect:** controller, `controller`
4. After raising PR, the reviewers will give feedback and comment, please address the comments or give suggestion if you think the comments don't make sense. Many people who are new to contribution thought PR is completion.
5. Proof read your translation before raising PR. Check for typos. Choice of wordings are debatable, but we don't tolerate typos.

### Top 5 Most Common Translated Words
- _cluster_ -> [klaster](https://kbbi.kemdikbud.go.id/entri/klaster)
- _resource_ -> sumber daya
- _tool_ -> alat
- _default_ -> bawaan
- _port_ -> [porta](https://kbbi.web.id/porta)

For complete words, read the glossary (link is being reworked), consult with [KBBI here](https://kbbi.web.id/), the one from [Kemendikbud](https://kbbi.kemdikbud.go.id/), and the [RSNI Glosarium from Ivan Lanin](https://github.com/jk8s/sig-docs-id-localization-how-tos/blob/master/resources/RSNI-glossarium.pdf) to understand how to translate various technical and Kubernetes-specific terms. (If you're still not sure about something, please don't hesitate to ask in [Slack channel](https://kubernetes.slack.com/messages/CJ1LUCUHM), we won't bite 😄)

Also, there is no special style guide for writing Bahasa Indonesia. In general we can follow the [English style guide](https://kubernetes.io/docs/contribute/style/style-guide/) with some addition to italicizing imported words.

Please commit with your translations! When you get comments from reviewers, please address them. We expect claimed pages to be translated within 2 weeks. If you find that you can't commit any more, let the maintainers know so they can assign the page to someone else.

## For maintainers
When someone volunteers, edit this issue with the username of the volunteer, and with the PR. Ex:

- [x] Init Kubernetes ID (@girikuncoro) #13822 

When PRs are merged, make sure to mark that page as completed!

# Work Status
(🏃 Work in Progress; 👀 In Review)

This is our fourth milestone (M4) that we aim to complete during the Kubernetes v1.19+v1.20 release cycle. We will mostly continue on translating remaining `task` pages, and other top visited pages.

## Translation of Concepts (1 page)
- [x] /docs/concepts/architecture/control-plane-node-communication (@irvifa) #22300

## Translation of Tutorials (9 pages)
- [x] /docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/ (@deryrahman) #22319
- [x] /docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/ (@deryrahman) #22408
- [x] /docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/ (@deryrahman) #22319
- [x] /docs/tutorials/kubernetes-basics/explore/explore-intro/ (@supirman ) #23700
- [x] /docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/ (@deryrahman) #22408
- [x] /docs/tutorials/kubernetes-basics/expose/expose-intro/ (@supirman ) #23802
- [x] /docs/tutorials/kubernetes-basics/update/update-intro/ (@habibrosyad ) #23907
- [x] /docs/tutorials/stateful-application/basic-stateful-set/ (@habibrosyad )  #23987
- [x] /docs/tutorials/stateless-application/expose-external-ip-address/ (@irvifa) #22390
- [x] /docs/tutorials/kubernetes-basics/scale/ (@supirman ) #23919

## Katacoda Scenarios (6 pages)
_Localizing [katacoda bootcamp scenarios](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios) is needed because they are used in the interactive tutorial sections above_
- [x] [kubernetes-bootcamp-scenarios/1](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/1) (@deryrahman) https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/pull/19
- [x] [kubernetes-bootcamp-scenarios/4](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/4) (@supirman ) https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/pull/21
- [x] [kubernetes-bootcamp-scenarios/5](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/5) (@habibrosyad)
- [x] [kubernetes-bootcamp-scenarios/6](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/6) (@habibrosyad)
- [x] [kubernetes-bootcamp-scenarios/7](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/7) (@habibrosyad)
- [x] [kubernetes-bootcamp-scenarios/8](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/8) (@habibrosyad)

## Translation of Setup (3 pages)
- [x] 👀 /docs/setup/learning-environment/minikube/ (@imrenagi) #22479
- [x]  /docs/setup/production-environment/tools/kubeadm/high-availability/ (@evanap) #22669
- [x] /docs/setup/production-environment/container-runtimes/ (@ariscahyadi ) #22320

## Translation of Tasks (28 pages)
- [x] /docs/tasks/administer-cluster/configure-upgrade-etcd (@Miftahunajat) #22733
- [ ] 🏃 /docs/tasks/administer-cluster/topology-manager (@irvifa )
- [x] /docs/tasks/administer-cluster/reserve-compute-resources (@girikuncoro) #24017
- [x] /docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace (@evanap) #23188
- [x] /docs/tasks/administer-cluster/highly-available-master (@ariscahyadi ) #22444
- [x] /docs/tasks/administer-cluster/sysctl-cluster (@ariscahyadi ) #22847
- [x] /docs/tasks/administer-cluster/network-policy-provider/calico-network-policy (@ariscahyadi) #22411
- [x] /docs/tasks/administer-cluster/dns-custom-nameservers (@ariscahyadi ) #22710
- [x] /docs/tasks/administer-cluster/namespaces (@sumodirjo) #22420
- [ ] 🏃 /docs/tasks/administer-cluster/cluster-management (@phanama )
- [x] /docs/tasks/administer-cluster/dns-debugging-resolution/ (@danninov ) #22907
- [x] /docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/ (@Miftahunajat ) #23821
- [x] /docs/tasks/run-application/force-delete-stateful-set-pod/ (@danninov) #22914
- [x] /docs/tasks/run-application/run-stateless-application-deployment/ (@irvifa) #22728
- [ ]  👀 /docs/tasks/run-application/run-replicated-stateful-application (@irvifa)
- [x] /docs/tasks/debug-application-cluster/resource-usage-monitoring/ (@ariscahyadi) #22785
- [x] /docs/tasks/debug-application-cluster/debug-application-introspection (@ariscahyadi ) #23165
- [x] /docs/tasks/tls/managing-tls-in-a-cluster/ (@imrenagi) #22312
- [x] /docs/tasks/configure-pod-container/assign-memory-resource/ (@ariscahyadi ) #23833
- [x] /docs/tasks/configure-pod-container/configure-volume-storage (@danninov) #22912
- [x] /docs/tasks/configure-pod-container/share-process-namespace (@irvifa) #22403
- [x] /docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity (@irvifa) #22481
- [x] /docs/tasks/configure-pod-container/static-pod (@wahyuoi) #22344
- [x] /docs/tasks/access-application-cluster/list-all-running-container-images/ (@ariscahyadi ) #22512
- [x] /docs/tasks/inject-data-application/distribute-credentials-secure (@danninov) #22924
- [x] /docs/tasks/manage-kubernetes-objects/kustomization (@girikuncoro) #22657
- [x] /docs/tasks/manage-daemon/rollback-daemon-set (@girikuncoro) #22804

## Translation of Others (3 pages)
- [x] /docs/reference/access-authn-authz/rbac/ (part 2) (@habibrosyad) #24019
- [x] /docs/contribute/participate/ (@ariscahyadi) #22355
- [x] /docs/contribute/suggest-improvements/ (@wahyuoi) #22345

## Extras
_Take note translated pages other than listed above in case any contributions_

*Note: This contribution template is inspired by Indonesian folks translating [reactjs doc](https://github.com/reactjs/id.reactjs.org/issues/1)*

Related Docs:
1. [RSNI glossary](https://github.com/jk8s/sig-docs-id-localization-how-tos/blob/master/resources/RSNI-glossarium.pdf)

/language id
/cc @kubernetes/sig-docs-id-reviews

**Comment by irvifa** [comment]
I'm taking docs/concepts/architecture/control-plane-node-communication

**Comment by imrenagi** [comment]
Im taking this /docs/setup/learning-environment/minikube/ @girikuncoro

**Comment by imrenagi** [comment]
Im taking this /docs/setup/learning-environment/minikube/

**Comment by ariscahyadi** [comment]
I'm working on this `/docs/setup/production-environment/container-runtimes/`

**Comment by girikuncoro** [comment]
I'm taking `/docs/tasks/manage-kubernetes-objects/kustomization`

**Comment by deryrahman** [comment]
I'm taking this `/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/`

**Comment by deryrahman** [comment]
Next, I'll take this `/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/`

**Comment by irvifa** [comment]
I'm taking `/docs/tutorials/stateless-application/expose-external-ip-address/`

**Comment by wahyuoi** [comment]
I'm taking ` /docs/tasks/configure-pod-container/static-pod`

**Comment by ariscahyadi** [comment]
Next, I'm working on this `/docs/contribute/participate/`

**Comment by wahyuoi** [comment]
I'm taking `/docs/contribute/suggest-improvements/`

**Comment by irvifa** [comment]
I'm taking `/docs/tasks/configure-pod-container/share-process-namespace`

**Comment by sumodirjo** [comment]
I'm taking `/docs/tasks/administer-cluster/namespaces`

**Comment by irvifa** [comment]
I'm taking ` /docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity`

**Comment by deryrahman** [comment]
I'm taking `/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/`

**Comment by ariscahyadi** [comment]
I am working on this `/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy`

**Comment by sumodirjo** [comment]
I'm taking `/docs/tasks/administer-cluster/cluster-management`

**Comment by ariscahyadi** [comment]
I'm working on this `/docs/tasks/administer-cluster/highly-available-master`

**Comment by irvifa** [comment]
I'm taking `/docs/tasks/run-application/run-stateless-application-deployment/`

**Comment by ariscahyadi** [comment]
I will take this `/docs/tasks/access-application-cluster/list-all-running-container-images/`

**Comment by danninov** [comment]
I'm taking `/docs/tasks/administer-cluster/dns-debugging-resolution/`

**Comment by evanap** [comment]
Hi! I'm taking `/docs/setup/production-environment/tools/kubeadm/high-availability/`

**Comment by ariscahyadi** [comment]
I will continue with this `/docs/tasks/administer-cluster/dns-custom-nameservers`

**Comment by girikuncoro** [comment]
I'm taking ` /docs/tasks/manage-daemon/rollback-daemon-set` next

**Comment by Miftahunajat** [comment]
I'm taking `/docs/tasks/administer-cluster/configure-upgrade-etcd`

**Comment by irvifa** [comment]
I'm taking `/docs/tasks/run-application/run-replicated-stateful-application`

**Comment by ariscahyadi** [comment]
I'm working on this `/docs/tasks/debug-application-cluster/resource-usage-monitoring/`

**Comment by girikuncoro** [comment]
I'm taking `/docs/tasks/administer-cluster/reserve-compute-resources` next

**Comment by ariscahyadi** [comment]
I am working on this `/docs/tasks/administer-cluster/sysctl-cluster`

**Comment by danninov** [comment]
I'm taking ` /docs/tasks/configure-pod-container/configure-volume-storage`

**Comment by danninov** [comment]
I'm taking  `/docs/tasks/run-application/force-delete-stateful-set-pod/`

**Comment by danninov** [comment]
I'm taking `/docs/tasks/inject-data-application/distribute-credentials-secure`

**Comment by danninov** [comment]
There is double task in the list: `run-stateless-application-deployment` @girikuncoro

**Comment by girikuncoro** [comment]
@danninov oops thanks for noticing this, I've removed it

**Comment by joshuabezaleel** [comment]
I'll take ` /docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/`.

**Comment by evanap** [comment]
Yo I'm taking `/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace` next

**Comment by ariscahyadi** [comment]
I will take this `/docs/tasks/debug-application-cluster/debug-application-introspection`

**Comment by supirman** [comment]
I would like to take `/docs/tutorials/kubernetes-basics/explore/explore-intro/`

**Comment by supirman** [comment]
I will take `/docs/tutorials/kubernetes-basics/expose/expose-intro/`

**Comment by irvifa** [comment]
@ariscahyadi is there any PR that already have lgtm label from your side?
I'd like to have the chance to review it as well. Thanks and good job!

**Comment by ariscahyadi** [comment]
> @ariscahyadi is there any PR that already have lgtm label from your side?
> I'd like to have the chance to review it as well. Thanks and good job!

Please help to review my own PR https://github.com/kubernetes/website/pull/22847 :)

Also this https://github.com/kubernetes/website/pull/23188 which I think you have already approved it, but I don't know why it is still not merged!

**Comment by irvifa** [comment]
@ariscahyadi I’m putting the issue on sig-docs slack for https://github.com/kubernetes/website/pull/23188

**Comment by Miftahunajat** [comment]
Hello, I would like to take `/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/` next.

**Comment by ariscahyadi** [comment]
> @ariscahyadi I’m putting the issue on sig-docs slack for #23188

Hi @irvifa, I saw this message "New changes are detected. LGTM label has been removed." I am not sure if it has any impact on it.
Or can we just try to add /lgtm label again? :)

**Comment by ariscahyadi** [comment]
I am working in this `/docs/tasks/configure-pod-container/assign-memory-resource/`

**Comment by girikuncoro** [comment]
@sumodirjo do you have update on `/docs/tasks/administer-cluster/cluster-management`?

**Comment by supirman** [comment]
@girikuncoro `/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/` was included in #22408, but the katacoda embed is broken since the katacoda sceneario is not translated yet but the embed use `data-katacoda-lang="id"`.

**Comment by girikuncoro** [comment]
@supirman thanks for the heads up, we'll translate the katacoda scenario quickly, I think it's fine if we let it be in broken state for short period

**Comment by habibrosyad** [comment]
I've submitted a PR for [kubernetes-bootcamp-scenarios/8](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/8)

**Comment by habibrosyad** [comment]
And this PR https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/pull/23 as well for [kubernetes-bootcamp-scenarios/7](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/7)

**Comment by habibrosyad** [comment]
And this PR katacoda-scenarios/kubernetes-bootcamp-scenarios#24 as well for [kubernetes-bootcamp-scenarios/6](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/6)

**Comment by habibrosyad** [comment]
And finally this PR katacoda-scenarios/kubernetes-bootcamp-scenarios#25 for [kubernetes-bootcamp-scenarios/5](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios/tree/master/5)

**Comment by habibrosyad** [comment]
I'm taking https://k8s.io/docs/tutorials/kubernetes-basics/update/update-intro/

**Comment by habibrosyad** [comment]
Also taking https://k8s.io/docs/tutorials/stateful-application/basic-stateful-set/

**Comment by supirman** [comment]
I'm taking unlisted `/docs/tutorials/kubernetes-basics/scale/` to complete Kubernetes Basic Tutorial

**Comment by girikuncoro** [comment]
@supirman thanks, I've added your page to the list

**Comment by habibrosyad** [comment]
I'm taking the last one `/docs/reference/access-authn-authz/rbac/ (part 2)`, but maybe it will be finished by the next week...

**Comment by girikuncoro** [comment]
We are moving to new milestone (M5) in #25025 and closing this one. Thanks everyone for your awesome contribution! 🎉 

/close
